src/main/resources/application.yml

```diff
@@ -35,12 +35,19 @@ management:
         time-to-live: 2000ms
       show-components: always
       show-details: always
+      probes:
+        enabled: true
+      group:
+        liveness:
+          include: livenessStateProbeIndicator
+        readiness:
+          include: readinessStateProbeIndicator
     info:
       cache:
         time-to-live: 2000ms
   health:
     elasticsearch:
-      enabled: false
+      enabled: true
```

Comment:
not sure why this was turned off, so turning on and I'll check it works in Delius AWS as well as Cloud Platform